pkg/types.go

```diff
@@ -10,7 +10,8 @@ const (
 	VizierServiceNamespaceEnvName = "VIZIER_CORE_NAMESPACE"
 	VizierService                 = "vizier-core"
 	VizierPort                    = "6789"
-	ManagerAddr                   = VizierService + ":" + VizierPort
+	// ManagerAddr                   = "10.111.12.89:6789"
```

Comment:
if not using this then it can be removed

Reply:
Fixed